### PR TITLE
LG-1081 - Updates validation for Issuer

### PR DESCRIPTION
**Pull request by amoose**
The flood gates were opened when we commented out the validations on Issuer

Completes: https://cm-jira.usa.gov/browse/LG-1081

This PR re-enables basic validation on Issuer:

Re-enables format validation on Issuer to block whitespace
Removes commented out and unused code around Issuer builder and parser
Removes unused specs and code

### CODE REVIEW

**Comment by amoose** [comment]
Still cleaning out some cruft. 🔪

**Comment by amoose** [comment]
I will follow up with more code coverage separately.

File: app/models/service_provider.rb [hunk 1]
```diff
@@ -72,7 +61,7 @@ def self.possible_attributes
     ]
     Hash[*possible.collect { |v| [v, v] }.flatten]
   end
-  # rubocop:ensable MethodLength
```
Comment (jmhooper) [inline app/models/service_provider.rb]: nice

Comment (amoose) [reply app/models/service_provider.rb]: 💃

**Comment by stevegsa** [comment]
the @amoose is back!

**Pull request merged**